docker/gitlab-content-ci/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM demisto/python3-deb:3.9.1.14969
+
+RUN apt-get update
+
+RUN apt-get install -y curl jq unzip openssh-client gcc libffi-dev libssl-dev git zip netcat wget python2 python-pip
+
+# Currently sh is linked to dash, linking it to /bin/bash instead
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+# Installing nvm, npm and node.js
+ENV NODE_VERSION 12.18.4
+RUN \
+    curl https://raw.githubusercontent.com/creationix/nvm/v0.38.0/install.sh | bash \
+    && export NVM_DIR="/root/.nvm" \
+    && export NODE_VERSION="12.18.4" \
+    && . $NVM_DIR/nvm.sh \
+    && nvm install $NODE_VERSION \
+    && nvm alias default $NODE_VERSION \
+    && nvm use default
+
+ENV NVM_DIR /root/.nvm
+ENV NODE_PATH $NVM_DIR/versions/node/v$NODE_VERSION/lib/node_modules
```

Comment:
Why are you explicitly setting this? Isn't the benefit of using nvm the flexibility? Shouldn't this stuff be set during job execution depending on what job and what repo is using this image?

Reply:
This is set for the default npm usage.
you still have to `source $NVM_DIR/nvm.sh` in the container to use nvm at all, so the NVM_DIR should be defined as installed.
The rest is for the default npm usage which will only work if you explicitly add `nvm use default` to your container.
It seemed more logical to add a default node version to the nvm instead of installing it every time in the build.

Reply:
> This is set for the default npm usage.
you still have to source $NVM_DIR/nvm.sh in the container to use nvm at all, so the NVM_DIR should be defined as installed.
I have no issue with setting an nvm root directory environment variable. I don't understand why you need to set the `NODE_PATH` variable.